tests/acceptance/run.sh

```diff
@@ -82,7 +82,7 @@ if [ -n "${EXPECTED_FAILURES_FILE}" ]; then
   # Check that every failed scenario is in the list of expected failures
   # Loop through the keys of the FAILED_SCENARIO_PATHS array (! does that)
   for FAILED_SCENARIO_PATH in "${!FAILED_SCENARIO_PATHS[@]}"; do
-    grep -x "${FAILED_SCENARIO_PATH}" "${EXPECTED_FAILURES_FILE}" >/dev/null
+    grep -v "[${FAILED_SCENARIO_PATH}]" "${EXPECTED_FAILURES_FILE}" >/dev/null
```

Comment:
The `[]` are special characters for grep. But we want to match them literally. They need to be escaped.
```suggestion
    grep "\[${FAILED_SCENARIO_PATH}\]" "${EXPECTED_FAILURES_FILE}" >/dev/null
```

And I am not sure why you put the `-v` option. That reverses the logic of the `grep` and returns all the lines that do not match the pattern.